dump/pldm_oem_cmds.cpp

```diff
@@ -105,11 +105,14 @@ void newFileAvailable(uint32_t dumpId, pldm_fileio_file_type pldmDumpType,
             Reason("Failed to open PLDM for new dump available request"));
```

Comment:
If retCode is not success, dont we have to return from here instead of attempting to do pldm_transport_send_msg() API ?

Reply:
elog<NotAllowed> this method will throw exception called NotAllowed so code will not continue further